### PR TITLE
refactor: navigation

### PR DESCRIPTION
**Pull request by nicolasbrugneaux**

### CODE REVIEW

File: src/components/list/row.tsx [hunk 1]
```diff
@@ -22,13 +23,13 @@ export const Row = (props: PropsWithChildren<Props>) => {
       <span className="truncate">{props.name}</span>
       <span className={`flex flex-initial flex-shrink-0 flex-row items-center gap-2`}>
         {props.children}
-        <a
+        <Link
```
Comment (aaronmgdr) [inline src/components/list/row.tsx:26]: hmm next Link should not be used for external links

Comment (nicolasbrugneaux) [reply src/components/list/row.tsx:26]: Shouldn't all rows be internal links anyway?

Comment (aaronmgdr) [reply src/components/list/row.tsx:26]: no sometimes we said these would link to an external website

Comment (nicolasbrugneaux) [reply src/components/list/row.tsx:26]: Hmm that's right, we'll need to adjust this then

Comment (aaronmgdr) [reply src/components/list/row.tsx:26]: i guess thats not MVP so we can change this when that happens

Comment (nicolasbrugneaux) [reply src/components/list/row.tsx:26]: Agreed

File: src/pages/validators.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import * as ga from '../utils/ga';
+
+const ValidatorsPage: NextPage = () => {
+  const router = useRouter();
+  const onModeChange = (mode: Mode) => {
+    void router.push({
+      pathname: `/${mode}`,
+    });
+    const handleRouteChange = (url: URL) => {
+      ga.pageview(url);
```
Comment (aaronmgdr) [inline src/pages/validators.tsx:14]: if we are using regular pages do we still need to record the page view or doesnt ga do that automatically?

Comment (nicolasbrugneaux) [reply src/pages/validators.tsx:14]: hmm should be automagical now

**Pull request merged**